modules/alloy/src/main/ts/ephox/alloy/dom/DomRender.ts

```diff
@@ -19,13 +22,36 @@ const renderToDom = (definition: DomDefinition.GeneralDefinitionDetail<SugarElem
     Value.set(subject as SugarElement<HTMLInputElement | HTMLTextAreaElement>, value);
   });
 
-  if (!definition.uid) {
-    // eslint-disable-next-line no-debugger
-    debugger;
+  return subject;
+};
+
+const attemptPatch = (definition: DomDefinition.GeneralDefinitionDetail<SugarElement<Node>>, obsoleted: SugarElement<Element>): Optional<SugarElement<Element>> => {
+  try {
+    const e = reconcileToDom(definition, obsoleted);
+    return Optional.some(e);
+  } catch (err) {
```

Comment:
It might be worth logging this error.

Reply:
That might be fine for development but we don't want that spamming in production if it were to happen which is why I didn't include a console log here.